openreview/openreview.py

```diff
@@ -200,7 +200,15 @@ def get_profile(self, email_or_id):
         profile = response.json()['profile']
         return Profile.from_json(profile)
 
-    def get_profiles(self, email_or_id_list, limit=1000):
+    def get_profiles_by_name(self, first, last):
+        '''
+        Returns a single profile by first and last name
```

Comment:
I think the doc-string needs to be corrected, since the function is returning a list and is called get_profiles_by_name